Ajout lors d'une nouvelle installation la possibilité de charger directement son backup.

lucguinchard opened the pull request:
Pour tester :
Installer des freshs dans tout les sens avec des backups et sans backup.

lucguinchard committed:
fix: syntax color

lucguinchard committed:
Ajout lors d'une nouvelle installation la possibilité de charger directement son backup.

Esso1-jeedom reviewed (approved):
si possible de prévoir une info attente pour le chargement . a l'appui du bouton  c tout .

lucguinchard committed:
Modification de l'icon lors de l'upload du fichier par une icon de chargement

lucguinchard commented:
@Esso1-jeedom J'ai fait une modification, même Branche même PR

lucguinchard committed:
suppression ligne dupliqué.

lucguinchard committed:
uniformisation des méthodes `restoreLocal` + `restoreBackupLocal`.
Cela permet de ne plus avoir le chemin complet ('/var/lib/nextdom/backup/') dans la liste des backups.
nettoyage du js.
méthode `SystemHelper::php()` avec un seul `exec()`

lucguinchard committed:
bugfix test unitaire : vérifie si le fichier existe et n'est pas un répertoire.

iWils reviewed (approved):
Test OK, mis à part un message d'erreur rencontrer dès l'étape 1 le firstUse est OK avec et sans backup.

lucguinchard committed:
Mise en place d'un log.js pour séparer l'affichage des logs + Ajout d'enums dans ConfigKey

iWils commented:
Suite à la mise à jour, il n'y a plu de pop-up erreur.
PR Ok pour moi
